### DISCUSSION TITLE
Set as... and quick editor.

### DISCUSSION THREAD

**Discussion by n-berenice** [deckerst/aves#186] [Q&A]
Is there any chance the app will have its own _Set as..._ feature and quick editor? Right now the Set as option simply redirects one to the default Android gallery and its own editor, if said gallery app is uninstalled then it won't be possible to set wallpapers anymore and Aves will crash unless another third party app that can handle crop and resize is installed.

This *could* be an issue native to Anx86 since the su access allows me to do things an user isn't supposed to but I'm not sure, I don't have any other devices to test other than an ancient Android 2.2 tablet that's not compatible with anything at this point.

**Comment by deckerst** [comment]
> Is there any chance the app will have its own Set as... feature and quick editor? 

Yes, but it's not a priority for now, because standard ROMs come with default apps that can already handle that.

In a way, an image editor is a completely different app independent of the gallery itself, that's why I'm currently focusing on the gallery. I'd rather offer a good gallery with no editor, than a mediocre gallery with a mediocre editor.

> Aves will crash

Thanks for heads-up! I thought the app was protected against this... but apparently not.
